gulpfile.js

```diff
@@ -168,8 +168,11 @@ gulp.task('build:scss', 'Build SCSS files', function () {
     var stream = gulp
       .src(pack.src)
       .pipe(sourcemaps.init())
-      .pipe(sass(pack.sassOptions).on('error', sass.logError))
-      .pipe(autoprefixer(pack.prefix))
+      .pipe(sass(pack.sassOptions).on('error', sass.logError));
+      if (pack.concat) {
```

Comment:
Not sure if this was intentional?  I'm not sure we need to support concatenation of SCSS files, since you can @import them from inside the file.